src/style.css

```diff
@@ -0,0 +1,119 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+  font-style: normal;
+}
+
+/* variables */
+:root {
+  --input-big-left-margin: 61px;
+  --input-small-left-margin: 32px;
```

Comment:
I have no reason to keep these values as variables

Reply:
is not removed. Or, you can tag me at fe_chat to give me a clue why do you need to use these variables